ddtrace/tracer/span.go

```diff
@@ -192,8 +192,11 @@ func takeStacktrace(n, skip uint) string {
 	return builder.String()
 }
 
-// setTagString sets a string tag. This method is not safe for concurrent use.
-func (s *span) setTagString(key, v string) {
+// setMeta sets a string tag. This method is not safe for concurrent use.
+func (s *span) setMeta(key, v string) {
+	if s.Meta == nil {
+		s.Meta = make(map[string]string, 1)
```

Comment:
Why initialize with a capacity of 1?

Reply:
Becase we only need one 🤷‍♂

Reply:
I have no idea honestly if this helps with anything... It might help in a tiny amount if the span doesn't have more than 1 tag, but then again, might do the opposite if it has 2.